### ISSUE TITLE
Training: variable scaling in loss

### ISSUE THREAD

**Issue by inakbk** [mateuszmatu/OceanAI#230]
Test variable scaling in the loss function. 

Try:
- [ ] scaling u & v differently than temp & salt
- [ ] scaling salt less 
- [ ]  removing zeta as forcing, and add scaling of ex. 0.1 in loss function
